src/main/java/xbot/common/injection/electrical_contract/IMUInfo.java

```diff
@@ -0,0 +1,6 @@
+package xbot.common.injection.electrical_contract;
+
+import xbot.common.controls.sensors.XGyro;
+
+public record IMUInfo(XGyro.InterfaceType interfaceType, CANBusId canBusId, int deviceId) {
```

Comment:
⭐  Seems like depending on if it's a CAN device or not the bus id isn't used, could we have 2 different constructors to handle these 2 cases so we don't have to throw a `null` in there and wonder what it's for?